src/Chapter1.hs

```diff
@@ -632,7 +641,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = last1 (abs n) + last2 (abs n)
+  where
+    last1 :: Int -> Int
+    last1 x = mod x 10
+    last2 :: Int -> Int
+    last2 x = div (mod x 100 - last1 x) 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod (abs n) 10
div (abs n) 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Thank you for suggestion! 
With `divMod` the solution will be:

```haskell
sumLast2 :: Int -> Int
sumLast2 n = fst x + snd x
  where
    x = divMod (mod (abs n) 100) 10
```

The `mod (abs n) 100` drops everything before the last 2 digits.
And `divMod ... 10` return the pair with of two last digits.

Reply:
That's a great solution! 👏🏻 

You can even use pattern-matching on the `x` variable but that's optional of course 🙂 

```haskell
sumLast2 :: Int -> Int
sumLast2 n = a + b
  where
    (a, b) = divMod (mod (abs n) 100) 10